python/python/bystro/examples/Demo_Learning_Predictive_Network.ipynb

```diff
@@ -0,0 +1,643 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Load in imports "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pickle\n",
+    "from sklearn.preprocessing import StandardScaler\n",
+    "from sklearn.decomposition import PCA\n",
+    "from sklearn.linear_model import LogisticRegressionCV\n",
+    "from sklearn.metrics import roc_auc_score,roc_curve\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "import torch\n",
+    "\n",
+    "import sys\n",
+    "sys.path.append('/home/austin/Forks/bystro/python/python/bystro/supervised_ppca/')\n",
+    "from gf_dropout_pt import PPCADropout"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Application: Neuroscience\n",
+    "\n",
+    "While we are in the genetics business, it's difficult to have an intuition of genes like ERBB2, MET, MYC, etc to evaluate whether the networks we uncover are reasonable. Instead, we will use a neuroscience application to demonstrate our ability to discover predictive networks. This data comes from 26 mice implanted with electrodes in 11 different brain regions. These mice are then placed in stressful and not stressful conditions while recording the voltages in these 11 regions (LFPs). \n",
+    "\n",
+    "Previous work has shown that spectral information in these LFPs is highly relevant biologically. We divide the recordings into 1-second windows and extract spectral power from 1 to 56 hertz in each region. This allows us to have both temporal and spectral resolution. After this process we have approximately 58k windows, each with 616 spectral power features, a behavioral label (stress vs not stress), and an index indicating which mouse the data came from.\n",
+    "\n",
+    "#### Data location:\n",
+    "\n",
+    "https://research.repository.duke.edu/concern/datasets/zc77sr31x?locale=en"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Load Dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "myDict = pickle.load(open('/media/austin/ThickBoy__1/DataTST/TST_Data.p','rb'))\n",
```

Comment:
Can't hardcode the path; please share this file and I will put it in a public repository

Reply:
I see that you provided where to download the files; please update the cell to add a comment that this should be downloaded from https://research.repository.duke.edu/downloads/kd17ct787?locale=en